lib/util/auth.js

```diff
@@ -42,10 +42,12 @@ module.exports = {
                     );
                     Util.logger.warn('Missing scope: ' + missingAccess.join(', '));
                 }
-                const existingAuth = await File.readJson(Util.authFileName);
+                const existingAuth = (await File.pathExists(Util.authFileName))
+                    ? await File.readJson(Util.authFileName)
+                    : {};
                 existingAuth[credential] = authObject;
 
-                File.writeJSON(Util.authFileName, existingAuth);
+                File.writeJSON(Util.authFileName, existingAuth, { spaces: '\t' });
```

Comment:
what for?

Reply:
cf you other comment about removing the file. this is where we save it, and this basically just pretty prints, otherwise its minified ;)